### PR TITLE
188. Log-Softmax & Cross-Entropy

### PR DESCRIPTION
**Pull request by preethamak**
Added a new problem. 188. Log-softmax and cross entropy in deep learning.

### CODE REVIEW

**Comment by preethamak** [comment]
@moe18 I have added this question
